frontend/src/concepts/projects/ProjectsContext.tsx

```diff
@@ -44,37 +46,51 @@ type ProjectsProviderProps = {
 };
 
 const ProjectsContextProvider: React.FC<ProjectsProviderProps> = ({ children }) => {
-  const fetchProjects = React.useCallback(() => getDSGProjects(), []);
+  const fetchProjects = React.useCallback(() => getProjects(), []);
```

Comment:
mmmm I think this is the simplest solution, but might induce into some performance issues, if a user just want to list ds projects, it will wait until all the projects are loaded, which is not ideal.
I get that is way easier filtering client side, so for me it's not a blocker and I can lgtm this, but it's something we might wanna take a look or maybe do a follow up in the future.